### ISSUE TITLE
MAS 1.4.11: Luminosity contrast ratio of the divider lines of the DataGridView control are less than 3:1

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#5876]
.NET Core Version: 
- 6.0.100-rc.1.21430.12

Have you experienced this same bug with .NET Framework?: 
- Yes

**Repro steps:**

1. Create a form with a `DataGridView`
2. Add multiple `Columns`
3. Add multiple `Rows`
4. Set `EnableHeadersVisualStyles = false`
5. Set the `DividerWidth = 5` for each Column
6. Set the `DividerHeight = 5` for each Row
7. Set `CellBorderStyle`
8. Set `ColumnHeadersBorderStyle`
9. Run application

**Actual behavior:**
- **DataGridViewCellBorderStyle.Sunken:**
```
dataGridView.CellBorderStyle = DataGridViewCellBorderStyle.Sunken;
dataGridView.ColumnHeadersBorderStyle = DataGridViewCellBorderStyle.Sunken;
```
![image](https://user-images.githubusercontent.com/23376742/135088316-8948bb2d-f126-4d9a-9049-d73921ce7b6d.png)
- **DataGridViewCellBorderStyle.Raised:**
```
dataGridView.CellBorderStyle = DataGridViewCellBorderStyle.Raised;
dataGridView.ColumnHeadersBorderStyle = DataGridViewCellBorderStyle.Raised;
```
![image](https://user-images.githubusercontent.com/23376742/135088996-7b700ebd-1cd3-42ce-bfba-8ab7f8380cca.png)
![image](https://user-images.githubusercontent.com/23376742/135089361-53b90797-6ea5-4713-982c-6139c643375b.png)
- **Single:**
```
dataGridView.CellBorderStyle = DataGridViewCellBorderStyle.Single;
dataGridView.ColumnHeadersBorderStyle = DataGridViewCellBorderStyle.Single;
```
![image](https://user-images.githubusercontent.com/23376742/135088996-7b700ebd-1cd3-42ce-bfba-8ab7f8380cca.png)
![image](https://user-images.githubusercontent.com/23376742/135089361-53b90797-6ea5-4713-982c-6139c643375b.png)

**Expected behavior:**
Luminosity contrast ratio of the divider lines of the `DataGridView` control are greater than 3:1.

**Comment by NikitaSemenovAkvelon** [comment]
It looks like the bug has been fixed already. 
1) Sunken (all cells have contrast borders): 
![image](https://user-images.githubusercontent.com/109065597/179736718-b62222da-8356-479d-bae6-5657ff9c2444.png)
2) Raised (like in previous case all cells have contrast borders): 
![image](https://user-images.githubusercontent.com/109065597/179737249-d0105237-7547-43f5-a092-244a126e479d.png)
3) Single (all borders have a same color):
![image](https://user-images.githubusercontent.com/109065597/179736393-623b6271-9f93-47c1-93ba-6effd5ea659d.png)

In summary, all cells in DataGridView separated from other by contrast borders. @merriemcgaw, what do you think if we close this issue as not reproduced?

**Comment by merriemcgaw** [comment]
I do think we should close this if it doesn't repro. I don't think it's a scenario that is all that common with the giant dividers, so if we're good with a fairly "normal" DGV I think we should close it.

**Comment by NikitaSemenovAkvelon** [comment]
The same ratio by a default style of DataGridView.
![image](https://user-images.githubusercontent.com/109065597/180161565-7867bbee-e789-4b25-a454-eddda4003d57.png)

**Comment by vladimir-krestov** [comment]
@merriemcgaw, @dreddy-work, so may we close this now?

**Comment by merriemcgaw** [comment]
Yup
